src/serialize.ts

```diff
@@ -48,6 +48,12 @@ export default function serialize(
     return `a:${serializeObject(item, scope)}`
   }
 
+  if (item instanceof Map) {
+    return `a:${item.size}:{` + Array.from(item.entries()).map(([value, key]) => {
+      return `${serialize(value, scope)}${serialize(key, scope)}`
+    }) + '}'
```

Comment:
Sorry guys, I just tried in my project. It seems we need an extra `.join('')` here because I am getting `Invalid serialized array format. Got: "a:2:{i:71654;i:61298;,i:71655;i:61299;}"` when there is more than 1 element in the array. I see the test is testing only the case with the one item in the array but for 2+ items there it fails because of extra `,`.